### ISSUE TITLE
Launchpad? Dashboard? Search field?

### ISSUE THREAD

**Issue by per-samuelsson** [StarcounterSamples/Launcher#166]
Hello there! I'm new to app development. I understand that a launcher is a good thing to have. Not that I totally understand its role, but its referred to in quite a lot of places.

On the repo landing page, I can read:

> This will bring you an empty Launcher (**with Launchpad, Dashboard and Search field**).

Thats generous of you! What do they do for me?

**Comment by warpech** [comment]
Good catch! I think the README.md files of all our sample apps should be improved.

Regarding Launcher, I wrote some information here: http://starcounter.io/guides/web/launcher/

I think all of it could be copied to README.md actually, do you agree?

**Comment by per-samuelsson** [comment]
> I think all of it could be copied to README.md actually, do you agree?

Without a doubt! And I was just about further harassing what was up with this "AppName" triplet I see on all the samples (now that I'm about to become an app developer myself), but hux flux, in that link you shared it was described too! :+1:

**Comment by per-samuelsson** [comment]
> I think all of it could be copied to README.md.

And oh, you want me to do it? I volonteer!

**Comment by tomalec** [comment]
If you volunteer, feel free to do so :) otherwise I can do so once I'm done with juicy-tile-editor

**Comment by per-samuelsson** [comment]
> If you volunteer, feel free to do so :)

Taken! Will do. Figured it could be my once-in-a-lifetime oppurtunity to sneak in a commit in the launcher project! :wink:

**Comment by per-samuelsson** [comment]
> Taken! Will do.

I discussed this some with @warpech. We considered different options, but decided on to keep the ReadMe remain more focused on building, packaging, etc. And to have the real content *about* launchers remain on the `.io` site.

What I ended up doing is to just copy-tweak the summary, and then provide a link back to `.io` 

[WDYT?](https://github.com/StarcounterSamples/Launcher#launcher)

For me, that solve the raised OP-issue. I'll close this one.
